docs/manual/_sources/manual_circuit.rst.txt

```diff
@@ -882,21 +882,21 @@ The primary use for tableaux in ``pytket`` is as a scalable means of specifying
 
 The data structure used here for tableaux is intended for compilation use. For fast simulation of Clifford circuits, we recommend using the :py:class:`StimBackend` from ``pytket-stim``, the :py:class:`SimplexBackend` from ``pytket-pysimplex`` (optimized for large sparse circuits), or the :py:class:`AerBackend` from ``pytket-qiskit``. Future versions of ``pytket`` may include improved decompositions from tableaux, as well as more flexible tableaux to represent stabilizer states, isometries, and diagonalisation circuits.
 
-Conditional Gates
-=================
+Classical and conditional operations
+====================================
 
 .. Performing gates conditional on some classical value or expressions
 
 Moving beyond toy circuit examples, many applications of quantum computing require looking at circuits as POVMs for extra expressivity, or introducing error-correcting schemes to reduce the effective noise. Each of these requires performing measurements mid-circuit and then performing subsequent gates conditional on the classical value of the measurement result.
 
 .. Specified by the `condition` kwarg when adding a gate
 
-Any ``pytket`` gate can be made conditional at the point of adding it to the :py:class:`Circuit` by providing the ``condition`` kwarg. The interpretation of ``circ.G(q, condition=reg[0])`` is: "if the  bit ``[reg[0]`` is set, then perform ``G(q)``".
+Any ``pytket`` operation can be made conditional at the point of adding it to the :py:class:`Circuit` by providing the ``condition`` kwarg. The interpretation of ``circ.G(q, condition=reg[0])`` is: "if the  bit ``reg[0]`` is set, then perform ``G(q)``".
```

Comment:
Could you clarify what is meant by the bit reg[0] being set? Does this mean that the operation is performed if reg[0] is assigned a bit value regardless of whether its 0 or 1?

Reply:
it means set to 1, I will clarify